### PR TITLE
[ValidatorSet] Prevent duplicated addresses among validators

### LINKED ISSUE #57: The candidates can share the same bridge operator and other data
We should not allow validators to share the same admin, treasury, or bridge operator

_Originally posted by @ducthotran2010 in https://github.com/axieinfinity/ronin-dpos-contracts/pull/22#discussion_r1005518493_
      https://github.com/axieinfinity/ronin-dpos-contracts/blob/603d7f6df260d2f86e9f90e8c5be536f933698cc/contracts/ronin/validator/CandidateManager.sol#L51-L67

### PR DESCRIPTION
**Pull request by nxqbao**
### Description

- https://skymavis.atlassian.net/browse/PSC-112?atlOrigin=eyJpIjoiMjRhN2Q3NmVlYmI4NDYzYzg3NzhmNmM3ZTY1N2JhZGEiLCJwIjoiaiJ9
- Fix #57 

### Checklist
- [x] I have clearly commented on all the main functions following the [NatSpec Format](https://docs.soliditylang.org/en/v0.8.0/natspec-format.html)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works

### CODE REVIEW

File: contracts/ronin/validator/CandidateManager.sol [hunk 1]
```diff
@@ -53,6 +54,46 @@ abstract contract CandidateManager is ICandidateManager, PercentageConsumer, Has
     require(!isValidatorCandidate(_consensusAddr), "CandidateManager: query for already existent candidate");
     require(_commissionRate <= _MAX_PERCENTAGE, "CandidateManager: invalid comission rate");
 
+    for (uint _i = 0; _i < _candidates.length; _i++) {
+      ValidatorCandidate storage existentInfo = _candidateInfo[_candidates[_i]];
```
Comment (ducthotran2010) [inline contracts/ronin/validator/CandidateManager.sol:58]: Should measure the gas used, IMO the apply-candidate tx should not be larger than 500k

Comment (nxqbao) [reply contracts/ronin/validator/CandidateManager.sol:58]: I have thought about it. There are two options:
1. Sacrifice execution gas one time per one validator. The later validators will need to spend more gas.
2. Need 4 more mappings on storage to support only this one function.

I'm more into option 1.

Comment (nxqbao) [reply contracts/ronin/validator/CandidateManager.sol:58]: In the worst case, there are 41 existent candidates, and `41 * 3 * 2100 ~= 260k` more gas is needed to query these values.

Comment (ducthotran2010) [reply contracts/ronin/validator/CandidateManager.sol:58]: I guess the number is acceptable

**Review by ducthotran2010 — APPROVED** [review]

**Pull request merged**